### ISSUE TITLE
Meetings: Recent meetings recordings are not showing on the upgrade test site

### ISSUE THREAD

**Issue by shrayshray** [datamade/la-metro-councilmatic#557]
The recordings of last week's committee meetings (2/18 and 2/19) are showing up on [boardagendas.metro.net](https://boardagendas.metro.net/events/) but not on [lametro-upgrade.datamade.us](https://lametro-upgrade.datamade.us/events/).

They are missing from both the meetings and events page, and the page of individual meetings. Example: https://lametro-upgrade.datamade.us/event/operations-safety-and-customer-experience-571116c9124b/

Meetings prior to last week are showing audio links as expected.

**Comment by hancush** [comment]
Looks like the audio came in. Will circle back to this one.

**Comment by hancush** [comment]
@shrayshray Exciting (?) update: From our conversation today, we observed that making event video public does not toggle the updated timestamp on events, so audio is only captured in full scrapes. 

Looking back over info on our end, I deployed a change on 2/26 that fixed a bug where the full scrape wasn't scraping anything for the upgrade instance. This would explain why event audio wasn't showing up on the upgrade site for 10 days, as of the time you opened this issue.

The first time the repaired full scrape would have run is the evening of the day you opened this issue, so it would also explain why the audio eventually came through.
